doc/api/next_api_changes/deprecations/24913-AL.rst

```diff
@@ -0,0 +1,4 @@
+``bbox.anchored()`` with no explicit container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If no *container* is passed to `.BboxBase.anchored`, the call simply returns a
+frozen copy of the original bbox; this is now deprecated.
```

Comment:
```suggestion
Not passing a *container* argument to `.BboxBase.anchored` is now deprecated.
```

Reply:
done

Reply:
Anyone should merge when green enough....